public/components/layer_control/layer_configuration_options.tsx

```diff
@@ -99,6 +113,28 @@ function LayerConfigurationOptions(props: MapsExlorerOptionsProps) {
         setValue={setLayerValue}
       />
 
+      <EuiSpacer size="s"/>
+
+      <EuiText size="xs">
+         <strong>
+         <FormattedMessage id="mapsExplorer.layerVisParams.zoomLevelLable" defaultMessage="Zoom Level*"/>
+           </strong>
+      </EuiText>
+
+      <EuiDualRange
+        min={0}
+        max={14}
+        step={1}
+        value={[stateParams.layersOptions[configLayerId].minZoom, stateParams.layersOptions[configLayerId].maxZoom]}
+        onChange={setZoom}
+        showLabels
+        showRange
+        // showInput
+        // showTicks
+        // fullWidth
```

Comment:
any reason for these comments?

Reply:
These are the props which can show the zoom value. Junqiu just showed me the latest Kibana UI(attached below), will update soon.
<img width="453" alt="image" src="https://user-images.githubusercontent.com/47327467/179634243-bd2a0ebf-54c4-4c13-a436-6a3675723254.png">